addons/o11y-definitions/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: o11y-definitions
-version: v0.3.0
+version: v0.3.1
```

Comment:
why upgrade this version while I didn't see any changes on the content?

Reply:
Yes, I added the `template.cue` file to the addon as the presence of this file is required by the vela cli addon command